### ISSUE TITLE
Global mappings wrapper/guards

### ISSUE THREAD

**Issue by amerlyq** [albfan/ag.vim#26]
See enhancement inside 81be8094f900ad8569a8d2fa752effbbc2138022.
Added `<Plug>` and guards for user mappings.

Planned: extend shortcuts list after #25.

**Comment by amerlyq** [comment]
Added outline for other `Ag*, LAg*` mappings in ebf90d10772b62903f41989b39edd028c8ef29a8.
They became activated automatically, when corresponding `<Plug>` mappings will be added.

**Comment by albfan** [comment]
What is left to do on this task?, go ahead and finish it.

If something is blocking it, open an issue and isolate it (labeling "blocked by" and "blocking") 

Divide and conquer

**Comment by amerlyq** [comment]
Alright, it was blocked, but it's due to fluffy workflow in beginning.
Now as concepts slightly changed, I postulate this work as done and all next upgrades will be separated. By the way, original purpose to keep this branch open was next:
- make draft for mappings to use
- wait until feature_args_derive merged
- fix draft by actual implementation -- (root of all evil)
- merge this and only than close

This way workflow could be completely parallel.
I still think that separate issue like "Fix primary assumed key mappings" is overkill.
But neither is good my previously described way -- at least for this project. Ideas?
